connector/src/main/java/org/geysermc/connector/network/translators/inventory/PlayerInventoryTranslator.java

```diff
@@ -97,7 +94,11 @@ public static void updateCraftingGrid(GeyserSession session, Inventory inventory
             slotPacket.setSlot(i + 27);
 
             if (session.getGameMode() == GameMode.CREATIVE) {
-                slotPacket.setItem(UNUSUABLE_CRAFTING_SPACE_BLOCK);
+                slotPacket.setItem(
+                        GeyserEdition.INVENTORY_UTILS.createUnusableSpaceBlock(
+                            "The creative crafting grid is\nunavailable in Java Edition"
+                        )
+                );
```

Comment:
This should stay how it was so we are not creating a new object every time

Reply:
That won't work as it was a static call before and thus would not exist at time the class is defined. The other option is to cache the creation on the first call and use the cached instance from then.

Reply:
Not sure why the static change was made, can you explain why you did this?

Reply:
UNUSUABLE_CRAFTING_SPACE_BLOCK was defined as:
```
private static final ItemData UNUSUABLE_CRAFTING_SPACE_BLOCK = GeyserEdition.INVENTORY_UTILS.createUnusableSpaceBlock(	
            "The creative crafting grid is\nunavailable in Java Edition");
```

However INVENTORY_UTILS does not exist at this time and I didn't want to impose a strict ordering on the dependency injection (thus the 2 stage regiser, then their setup methods are called).

So this means ditching the static definition in favour of a cached call like in the 2 resolved discussions that looks it up on first call.

Reply:
Ah yea, sorry missed a few things. Just make it cache the output on a static block and it should be fine. Message me on Discord if you're still confused

Reply:
Static block won't work either for the same reasons (it is executed when the .class file is read and will occur before the GeyserEdition has finished its registration). Best option would be a `getUnusableCraftSpaceBlock` that looks it up once and caches the value from then.

Reply:
Its run once when an object of that type is created or accessed which should be fine in this case?

Reply:
You mean like this:

https://github.com/bundabrg/Geyser/blob/a0a06655064341291d59ab8aa2e0b5df555f0799/connector/src/main/java/org/geysermc/connector/edition/mcpe/v1_14_60/Edition.java#L409

Will trigger the static and thats half way through all the registrations.

Reply:
Ah yes, resolving the conversation.

Reply:
I think all of this is beyond the scope of this PR